src/architecture/clients.md

```diff
@@ -38,12 +39,15 @@ There are two typical use cases for the light client:
 In this and next section all functions indicated in *italics* are the functions
 implemented in Exonum light client.
 
-1. Frontend triggered by an event (for example, a button click handler)
-  decides to create a new transaction
-2. The transaction data is stored in JSON format. Then the data is
-  *converted to the Exonum binary format* and *digitally signed*
-3. The generated transaction (JSON data + digital signature) is sent to the
-  a full node via an HTTP POST request
+![Send data to the blockchain](../images/send-data.png)
+
+1. Frontend triggered by an event (for example, a button click handler) decides to create a new transaction.
+  The transaction data is stored in JSON format.
+  Then the data is *converted to the Exonum binary format* and *digitally signed* using the light client.
+2. Frontend receives a transaction digital signature from the light client.
+3. The generated transaction (JSON data + digital signature) is sent to the a full node via an HTTP POST request.
+4. Frontend receives a transaction hash or another type of notification
+  when the transaction is successfully added to the block
```

Comment:
This isn't correct, at least for now. The frontend receives a notification (e.g., the transaction hash) as a response to the HTTP request, i.e., not waiting while the transaction is added to a block.